rule/struct-tag.go

```diff
@@ -275,6 +282,58 @@ func (lintStructTagRule) typeValueMatch(t ast.Expr, val string) bool {
 	return typeMatches
 }
 
+func (w lintStructTagRule) checkProtobufTag(tag *structtag.Tag) (string, bool) {
+	// check name
+	switch tag.Name {
+	case "bytes", "fixed32", "fixed64", "group", "varint", "zigzag32", "zigzag64":
+		// do nothing
+	default:
+		return fmt.Sprintf("invalid protobuf tag name '%s'", tag.Name), false
+	}
+
+	// check options
+	seenOptions := map[string]bool{}
+	for _, opt := range tag.Options {
+		if _, err := strconv.Atoi(opt); err == nil {
+			_, alreadySeen := w.usedTagNbr[opt]
+			fmt.Printf("opt %s already seen? %v\n", opt, alreadySeen)
+			if alreadySeen {
+				return fmt.Sprintf("duplicated tag number %s", opt), false
+			}
+			w.usedTagNbr[opt] = true
```

Comment:
014 and 14 will NOT regard as duplicate tag number since we use raw string as key, but I think no one will do that.

Reply:
I've modified the code to use ints as key